specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2021-03-01-preview/examples/entityQueryTemplates/GetActivityEntityQueryTemplateById.json

```diff
@@ -0,0 +1,58 @@
+{
+  "parameters": {
+    "api-version": "2021-03-01-preview",
+    "subscriptionId": "d0cfe6b2-9ac0-4464-9919-dccaee2e48c0",
+    "resourceGroupName": "myRg",
+    "workspaceName": "myWorkspace",
+    "operationalInsightsResourceProvider": "Microsoft.OperationalIinsights",
+    "entityQueryTemplateId": "07da3cc8-c8ad-4710-a44e-334cdcb7882b"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/d0cfe6b2-9ac0-4464-9919-dccaee2e48c0/resourceGroups/myRg/providers/Microsoft.OperationalInsights/workspaces/myWorkspace/providers/Microsoft.SecurityInsights/entityQueryTemplates/07da3cc8-c8ad-4710-a44e-334cdcb7882b",
+        "name": "07da3cc8-c8ad-4710-a44e-334cdcb7882b",
+        "type": "Microsoft.SecurityInsights/entityQueryTemplate",
+        "kind": "Activity",
```

Comment:
>kind [](http://example.com/codeflow?start=9&length=4)

kind is a reserved top level property. More details here : 

[https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md)

Reply:
what is the problem with this example?
when performing "get by id" request, the return resource is of kind "Activity".

Reply:
@raosuhas - any updates regarding this?

Reply:
@raosuhas - any updates?
Thank you